doc/source/python/python_wrapping_s2i.md

```diff
@@ -139,6 +139,20 @@ Using s2i you can build directly from a git repo or from a local source folder.
 s2i build <src-folder> seldonio/seldon-core-s2i-python3:1.2.3-dev <my-image-name>
 ```
 
+Additional Environment Flags:
```

Comment:
To align the docs structure, should we move this section to the one below (in the same page) titled "Environment Variables"?

Reply:
I was not sure if we want to put this in that section as these parameter are mostly used togather and that section gives an impression of variables need to be defined in s2i environment file.

We should not add EXTRA_INDEX_URL (with auth token) in the file, better read it from env.

Other thing that we can recommend in documentation is to pass it with s2i command rather than checking in in the file.

Reply:
That's a good point. I think both options are covered on that section. We can always add a note under this variable to recommend users to provide it directly as a CLI flag for the reasons you mention. 

I think that having all env vars there, under the same section, would make it easier to understand for the user. Otherwise, we would also need to explain why there are 2 separate env vars section and what's the difference between them.

Reply:
Hi @adriangonz, have updated documentation, kindly review.